feature-proxy-api/src/main/java/io/novafoundation/nova/feature_proxy_api/domain/model/ProxyType.kt

```diff
@@ -1,24 +1,24 @@
 package io.novafoundation.nova.feature_proxy_api.domain.model
 
-sealed class ProxyType(val name: String) {
+sealed class ProxyType(val name: String, val controllableFrom: List<ProxyType>) {
```

Comment:
Ideally this shouldnt be embedded into ProxyType itself since it is actually what `ProxtCallFilter` is responsible for. Refactoring it will require some efforts though so we can leave it as it is rn